api/api/api.py

```diff
@@ -0,0 +1,11 @@
+from ninja import NinjaAPI
+from django.urls import path
+
+api = NinjaAPI()
+
+api.add_router("/", "documents.views.document_router", tags=["Documents"])
+api.add_router("/", "chunks.views.chunk_router", tags=["Chunks"])
```

Comment:
**nitpick:** please tag routers at their level, not on the api level (router accepts `tags` parameter)